scripts/results/arrange-data.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-2019, Regents of the University of Arizona.
+# Author:Wenkai Zheng<wenkaizheng@email.arizona.edu>
+# DESCRIPTION:
+# ------------
+# This file is used for arranging data and print out plot
+# to run it you can have python arrange-data.py parameter(such as jitter, rebuffers or rtt)
+# for stdin you have 3 options
+# 1. no input: just left blank
+# 2. one date as input: month/day/year hour:minute:second for example Nov/4/2019 18:55:59
+# 3. two date as input: month/day/year hour:minute:second - month/day/year hour:minute:second for example Nov/4/2019 18:55:59 - Dec/3/2019 18:55:59 
+
+import datetime
+import sys
+import subprocess
+import os
+
+def trunc_datetime(line):
+
+    # DEFINITIONS START
+
+    global Months
+
+    # DEFINITIONS END
+
+    data = line.split()
+    year = data[4]
+    month = Months[data[1]]
+    day = data[2]
+    hour = data[3].split(':')[0]
+    minute = data[3].split(':')[1]
+    second = data[3].split(':')[2]
+    return datetime.datetime(
+        int(year),
+        int(month),
+        int(day),
+        int(hour),
+        int(minute),
+        int(second),
+        )

```

Comment:
No tailing parentheses or brackets (move them to the previous line). Go over the entire file and do the same.